linux/ibus-keyman/tests/scripts/test-helper.sh

```diff
@@ -9,3 +9,83 @@ function can_run_wayland() {
     return 0
   fi
 }
+
+function generate_kmpjson() {
+  local TESTDIR
+  TESTDIR="$1"
+  pushd "$TESTDIR" > /dev/null || exit
+  KMPFILE="${TESTDIR}/kmp.json"
+  THISYEAR=$(date "+%Y")
+  cat <<-EOF > "$KMPFILE"
+  {
+    "system": {
+      "keymanDeveloperVersion": "10.0.1099.0",
+      "fileVersion": "7.0"
+    },
+    "options": {
+      "readmeFile": "kmx.cpp"
+    },
+    "info": {
+      "name": {
+        "description": "Test Keyboards"
+      },
+      "version": {
+        "description": "0.0"
+      },
+      "copyright": {
+        "description": "\u00A9 2018-${THISYEAR} SIL International"
```

Comment:
I'd probably leave the copyright field out -- it's unimportant for a test

Reply:
Done in #8732.